app.py

```
@@ -21,8 +21,25 @@
 )
 logger.addHandler(stdout_handler)
 
-
-def get_repository_technologies(ql, org, batch_size=30):
+def find_keywords_in_file(file, keywords_list):
+    """Find keywords in a file
+
+    Args:
+        file (str): File contents that you are interested in searching
+        keywords_list (list): List of keywords that you want to search through the file
+
+    Returns:
+        list: List of keywords that were found in the file
+    """    
+    keywords = []
+    if file is not None:
```

I would consider un-nesting this

```python

if file is None:
    return []

for keyword in keywords_list:
    ...
```

Perhaps also consider raising an error if a file is not found instead of returning an empty list.

---

We wouldn't want to raise an error here as if keywords aren't found we would want the list to be empty. This mitigates us having to write a bunch of try-catches.